src/app/organizations/state/collections.service.ts

```diff
@@ -20,7 +20,6 @@ import { Router } from '@angular/router';
 import { ID, transaction } from '@datorama/akita';
 import { finalize } from 'rxjs/operators';
 import { AlertService } from '../../shared/alert/state/alert.service';
-import { OrganizationsService } from '../../shared/swagger';
 import { Collection, OrganizationsService as OpenApiOrganizationsService } from '../../shared/openapi';
```

Comment:
Don't think you need the `as OpenApiOrganizationsService` since there's no swagger conflict